draft-ietf-oauth-resource-metadata.xml

```diff
@@ -1870,9 +1877,17 @@
 	<list style="symbols">
 	  <t>
 	    Incorporated responses to HttpDir review comments by Mike Bishop.
-    </t>
-    <t>
-      Incorporated responses to IESG review comments by Roman Danyliw.
+	  </t>
+	  <t>
+	    Incorporated responses to IESG review comments by Roman Danyliw.
+	  </t>
+	  <t>
+	    Incorporated responses to IESG review comments by Orie Steele.
```

Comment:
It's probably worth calling out specifically the change that was made based on this review here

Reply:
Done in https://github.com/oauth-wg/draft-ietf-oauth-resource-metadata/pull/59/commits/5f9a7fd5bb9db63580308a130598c69dd576eb86